### PR TITLE
[DF] Add cumulative efficiency to cutflow report

### PR DESCRIPTION
**Pull request by dpiparo**
Implement the cumulative efficiency in the cutflow report as suggested at the ATLAS newcomers event.

### CODE REVIEW

**Comment by stwunsch** [comment]
Very nice! That's how it looks for the dimuon example:

```
Events with exactly two muons: pass=33370298   all=66128870   -- eff=50.46 % cumulative eff=50.46 %
Muons with opposite charge: pass=25794885   all=33370298   -- eff=77.30 % cumulative eff=39.01 %
```

Only missing piece: Adapting the test expectation to the new output strings.

File: tree/dataframe/inc/ROOT/RDF/RCutFlowReport.hxx [hunk 1]
```diff
@@ -41,7 +41,7 @@ public:
    const std::string &GetName() const { return fName; }
    ULong64_t GetAll() const { return fAll; }
    ULong64_t GetPass() const { return fPass; }
-   float GetEff() const { return 100.f * (fPass / float(fAll)); }
+   float GetEff() const { return 100.f * (float(fPass) / float(fAll)); }
```
Comment (bluehood) [inline tree/dataframe/inc/ROOT/RDF/RCutFlowReport.hxx]: i'm not sure this does anything, godbolt seems to [confirm](https://godbolt.org/z/dDaq-5)

Comment (dpiparo) [reply tree/dataframe/inc/ROOT/RDF/RCutFlowReport.hxx]: You are right :) !

**Comment by dpiparo** [comment]
Dear @stwunsch , @bluehood ,
thanks! I will apply the changes asap and merge!

**Pull request merged**